app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -3,51 +3,50 @@
   <% @seen_summary = {} %>
   <% @creations.each_with_index do |creation, index| %>
 
-    <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
+<% this_work = creation.respond_to?(:work) ? creation.work : creation %>
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title.html_safe %>
+<% if creation.is_a?(Work) then
```

Comment:
We don't use `then` in Ruby. In fact, this is a lot of Ruby for a view file -- it would really help readability if this big block were a helper method in, say, mailer_helper.rb, rather than all being here in the view file.